src/services/news2/service/news.service.js

```diff
@@ -0,0 +1,48 @@
+const { BadRequest } = require('@feathersjs/errors');
+const logger = require('../../../logger');
```

Comment:
Codacy found an issue: ['logger' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=6009087)